### ISSUE TITLE
Copy command in menu toolbar "Edit" menu doesn't work for multiple editors

### ISSUE THREAD

**Issue by ryan-stackwave** [TeselaGen/tg-oss#24]
@tnrich

The implementation of the copy command for the menu toolbar assumes there is only one element with class `veVectorInteractionWrapper` so copying via the "Edit" menu doesn't work for the second editor on a page:
https://github.com/TeselaGen/tg-oss/blob/master/packages/ove/src/commands/index.js#L362

**Comment by ryan-stackwave** [comment]
Sure @tnrich, I'll see what I can do.

**Comment by ryan-stackwave** [comment]
@tnrich, bad news. It looks like I can't upgrade to this version of ove yet because react 18 conflicts with another library I'm using. I'll have to look at this again when I upgrade react and work around in the meantime.

**Comment by ryan-stackwave** [comment]
Sorry for the delay @tnrich, got caught up in other priorities. I found the issue with using react 16 with ove and it seems to be because I'm using webpack. I was able to workaround it by specifying aliases in my webpack.js file to force all libraries to use my chosen version of react:
```
alias: {  
    react: path.resolve(__dirname, "./node_modules/react"),  
    "react-dom": path.resolve(__dirname, "./node_modules/react-dom"),  
    "react-redux": path.resolve(__dirname, "./node_modules/react-redux")  
}
```
This [stackoverflow post](https://stackoverflow.com/questions/66488492/solve-having-more-than-one-copy-of-react-in-the-same-app) seems to indicate that changing ove to make the react libraries peerDependencies rather than dependencies might make it more compatible with webpack but I haven't tried it yet.
